1.Exploratory_Data_Analysis/scripts/9.subset_umap.py

```diff
@@ -63,7 +63,13 @@
 anova_results.head()
 
 
-# In[7]:
+# Where
+# a_h = apoptosis vs healthy
+# a_p = apoptosis vs pyroptosis
+# h_p = healthy vs pyroptosis
+#
+
+# In[ ]:
```

Comment:
Please confirm sequential execution on all notebooks - I will stop making this comment here on!